docs/mllib-dimensionality-reduction.md

```diff
@@ -84,6 +84,25 @@ quick-start guide. Be sure to also include *spark-mllib* to your build file as
 a dependency.
 
 </div>
+<div data-lang="python" markdown="1">
+{% highlight python %}
+from pyspark.mllib.linalg.distributed import RowMatrix
```

Comment:
Now days we tend write new examples separately and then use the include example syntax to bring them